library/src/blas1/rocblas_copy_strided_batched.cpp

```diff
@@ -0,0 +1,187 @@
+        = "rocblas_hcopy_strided_batched";
+    template <>
+    constexpr char rocblas_copy_strided_batched_name<rocblas_float_complex>[]
+        = "rocblas_ccopy_strided_batched";
+    template <>
+    constexpr char rocblas_copy_strided_batched_name<rocblas_double_complex>[]
+        = "rocblas_zcopy_strided_batched";
+
+    template <rocblas_int NB, typename T>
+    rocblas_status rocblas_copy_strided_batched_impl(rocblas_handle handle,
+                                                     rocblas_int    n,
+                                                     const T*       x,
+                                                     rocblas_int    incx,
+                                                     rocblas_stride stridex,
+                                                     T*             y,
+                                                     rocblas_int    incy,
+                                                     rocblas_stride stridey,
+                                                     rocblas_int    batch_count)
+    {
+        if(!handle)
+            return rocblas_status_invalid_handle;
+
+        auto layer_mode = handle->layer_mode;
+        if(layer_mode & rocblas_layer_mode_log_trace)
+            log_trace(handle,
+                      rocblas_copy_strided_batched_name<T>,
+                      n,
+                      x,
+                      incx,
+                      stridex,
+                      y,
+                      incy,
+                      stridey,
+                      batch_count);
+
+        if(layer_mode & rocblas_layer_mode_log_bench)
+            log_bench(handle,
+                      "./rocblas-bench -f copy_strided_batched -r",
+                      rocblas_precision_string<T>,
+                      "-n",
+                      n,
+                      "--incx",
+                      incx,
+                      "--stridex",
+                      stridex,
+                      "--incy",
+                      incy,
+                      "--stridey",
+                      stridey,
+                      "--batch_count",
+                      batch_count);
+
+        if(layer_mode & rocblas_layer_mode_log_profile)
+            log_profile(handle,
+                        rocblas_copy_strided_batched_name<T>,
+                        "N",
+                        n,
+                        "incx",
+                        incx,
+                        "stridex",
+                        stridex,
+                        "incy",
+                        incy,
+                        "stridey",
+                        stridey,
+                        "batch_count",
+                        batch_count);
+
+        if(!x || !y)
+            return rocblas_status_invalid_pointer;
+
+        RETURN_ZERO_DEVICE_MEMORY_SIZE_IF_QUERIED(handle);
+
+        if(!incx || !incy || batch_count < 0)
+            return rocblas_status_invalid_size;
```

Comment:
`incx==0 || incy==0` should be okay. http://www.netlib.org/lapack/lapack-3.1.1/html/dcopy.f.html